src/model/configuration_example/model/falcon-7b-from-pretrained.yaml

```diff
@@ -17,3 +17,18 @@ args:
 tokenizer:
   pretrained_model_name_or_path: tiiuae/falcon-7b
   tokenizer_class: AutoTokenizer
+tokenized_dataset: None
```

Comment:
We should set the dataset config in `example_config.yaml` file.

Reply:
done